amundsen_application/static/js/components/ColumnList/ColumnStats/index.tsx

```diff
@@ -23,10 +24,12 @@ const ColumnStatRow: React.FC<ColumnStatRowProps> = ({
   stat_type,
   stat_val,
 }: ColumnStatRowProps) => {
+  const numberVal = +stat_val;
+
   return (
     <div className="column-stat-row">
       <div className="stat-name body-3">{stat_type.toUpperCase()}</div>
```

Comment:
Could you change this to use the new typography extends?
You can use this as an example: https://github.com/amundsen-io/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/components/ColumnList/styles.scss#L84